src/hooks/useWidgetWidth.ts

```diff
@@ -0,0 +1,10 @@
+import { WidgetWidthContext } from 'components/WidgetWrapper'
+import { useContext } from 'react'
+
+/**
+ * Gets the current width of the widget.
+ */
+export default function useWidgetWidth(): number {
```

Comment:
I don't think this is actually used. Consider instead just exporting a hook called eg `useIsWidgetNarrow` or `useIsWidgetWide`, or even returning an enum here that just defines `WIDE` or `NARROW`.